frontend/src/assets/style/_variables.scss

```diff
@@ -35,15 +39,21 @@
   --preparePageImportUserStoriesButton: #30a858;
   --preparePageActiveCardSet: black;
   --preparePageInActiveCardSet: #e27712;
-  --preparePageTimerBackground: #974f0d3d;
+  --preparePageTimerBackground: rgba(192, 175, 156, 0.4);
+  --preparePageNotSelectedTabBackground: rgba(192, 175, 156, 0.4);
 
   /*Session Page*/
-  --textAreaColour: #c3c7bd;
+  --textAreaColour: rgba(192, 175, 156, 0.4);
```

Comment:
Should be replaced by basic SCSS variable which is used for input fields, I think.